tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py

```diff
@@ -54,6 +54,8 @@ def set_func_attr(func, compile_name, symbol_name):
 )
 def test_conv2d(dpu_target):
```

Comment:
Can you get rid of `skip_test` and use pytest’s skipIf decorator so the skips are properly reported?

https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1

Reply:
@driazati 
I replaced skip_test with pytest's skipif. Could you have a another look?

Reply:
looks good, thanks!